README.md

```diff
@@ -1,3 +1,70 @@
-# field_monitoring
+![](RackMultipart20220705-1-ruigrg_html_70e3b217e6f91606.png)
+
+# ATLAS Service Template Specification
+
+# Name: Crop Status Monitoring
+
+# Author(s): AgroApps (AGA)
+
+_The &quot;__Crop Status Monitoring&quot; ATLAS Service Template_
+
+This service will deliver a number of EO derived indicators for crop status monitoring.
+
+## INTRODUCTION
+
+The **Crop Status Monitoring (CSM) Service**, is an earth observation crop monitoring module that provides crop monitoring and farming service products, through the use of validated algorithms. The aim of the **CSM service** is to provide farmers with information on their crops&#39; health status, through a biophysical proxy. Another aim is to provide estimations of the expected yield and crop residues. The CSM service can help farmers identify anomalies in plant performance during the growing season.
```

Comment:
You must not short "Crop Status Monitoring Service (CSM) Template" to "Crop Status Monitoring (CSM) Service"  (or CSM Service instead of CSM Service Template) as this causes confusion. A service is a vendor-specific implementation (i.e. based on EO), whereas a service template is an implementation-agnositic API definition. Better to be systematically wordy than to introduce ambiguity. See the terminology section in other service templates.